docs/install.md

```diff
@@ -47,7 +47,7 @@ You should set these as required before executing the scripts below.
 This step only has to be done once.
 
 ```shell
-git submodule update --init
+git submodule update --init --recursive
```

Comment:
There are at least two other places in the `docs` directory where the `--recursive` is missing.